src/pages/Recovery/Waterman.tsx

```diff
@@ -36,20 +42,61 @@ export const Waterman = () => {
       accountNumber,
       accountType,
     })
-    const sendMax = true
-    const fees = await bitcoinChainAdapter.getFeeData({
+
+    const { data: rawFees } = await axios.get(
+      `${getConfig().REACT_APP_UNCHAINED_BITCOIN_HTTP_URL}/api/v1/fees`,
+    )
+    console.log({ rawFees })
+
+    const utxos = [
+      // {
+      //   txid: '994929ee81f6909e53cacda3c39e16c45a0508fb8783f1d05c3fc4dfd8d5255a',
+      //   vout: 0,
+      //   value: '1000',
+      //   height: 768701,
+      //   confirmations: 4892,
+      //   address: 'bc1q9hz8dl4hhz6el7rgt25tu7rest4yacq4fwyknx',
+      //   path: "m/84'/0'/0'/0/0",
+      // },
+      {
+        txid: '2267f89589c7ce8c9be466658b021f998391856d66864fd951485c09d51d9d95',
+        vout: 0,
+        value: '33106961',
+        height: 768689,
+        confirmations: 4904,
+        address: 'bc1q9hz8dl4hhz6el7rgt25tu7rest4yacq4fwyknx',
+        // path: "m/84'/0'/0'/0/0", // this is per blockbook, but is not correct, or perhaps required?
+      },
+    ]
+
+    const satoshiPerByte = String(Math.round(rawFees.fast.satsPerKiloByte / 1024))
+    console.log({ satoshiPerByte })
+
+    const utxoSelectResult = utxoSelect.utxoSelect({
+      from: affectedAddress,
```

Comment:
Curious if we need `from` ? In the context of recovering funds, that's probably fine since funds are most likely in one address, but either way, since we manually build and pass UTXOs, letting coinselect use all UTXOs (e.g one) should be fine.

Though that's probably safer, given this sends back change to the same address so if for any reason there is change, that would make it easier to build and broadcast a second Tx.